deployed-mainnet.json

```diff
@@ -12,12 +11,6 @@
   "miniMeTokenFactoryAddress": "0x909d05f384d0663ed4be59863815ab43b4f347ec",
   "aragonIDAddress": "0x546aa2eae2514494eeadb7bbb35243348983c59d",
   "multisigAddress": "0x55Bc991b2edF3DDb4c520B222bE4F378418ff0fA",
-  "daoTemplateDeployTx": "0xdcd4ebe028aa3663a1fe8bbc92ae8489045e29d2a6ef5284083d9be5c3fa5f19",
-  "lidoBaseDeployTx": "0xd01afeac8b9231d6d18a874c17ad3995c8702964ade5285977681e69a588f687",
-  "oracleBaseDeployTx": "0xd16e53c87b643ba68cda34e261dda7a25ebb0b492d835a03f32fa48d5ebe68cd",
-  "nodeOperatorsRegistryBaseDeployTx": "0x32a2b4ec8b59f2cdde007bbdee23ed1b75491f0ca6acc25d45bf7afbf1c8b4dd",
-  "depositorDeployTx": "0xce40c18b10854166eb69e64171cbd33ec4ba2f369623f2f438bcf684cfed0523",
-  "executionLayerRewardsVaultDeployTx": "0xd72cf25e4a5fe3677b6f9b2ae13771e02ad66f8d2419f333bb8bde3147bd4294",
```

Comment:
it doesn't look good to remove deploy txs, I think it's better to add the missing ones

Reply:
What's the point of storing them? They are not used anywhere in the code and they are easily retrievable via Etherscan?

Reply:
Added because need them to generate artifacts.json